### PR TITLE
Improve our documentation about contributing documentation

### PR DESCRIPTION
**Pull request by kolloch**

### CODE REVIEW

File: docs/docs/contributing.md [hunk 1]
```diff
@@ -5,7 +5,43 @@ title: Contributor Guidelines
...
+generated from what is found in the docs directory.
+
+If you simply want to correct a spelling mistake or improve the wording of a sentence, you can browse
+the mark down files [here](https://github.com/mesosphere/marathon/tree/master/docs) and use the edit
+button above the markup. That will make it easy to create a pull request that will be reviewed by us.
+
+If want to contribute a larger improvement to our documentation: 
+
+* edit the files in the docs directory
+* check the rendered the result as described in the
```
Comment (gkleiman) [inline docs/docs/contributing.md]: s/the result/result/

I would capitalise the beginning of this bullet points, or make the following one start with lower case =).

**Comment by kolloch** [comment]
Thanks for the review. I changed the docs accordingly.

File: docs/docs/contributing.md [hunk 2]
```diff
@@ -5,7 +5,43 @@ title: Contributor Guidelines
...
+* Edit the files in the docs directory.
+* Check the rendered result as described in the 
+  [docs/README.md](https://github.com/mesosphere/marathon/blob/master/docs/README.md).
+* If you are feeling perfectionistic, check if there is already an issue about this documentation deficiency
+  [here](https://github.com/mesosphere/marathon/issues?q=is%3Aopen+is%3Aissue+label%3Adocs).
+  Prefix your commit message with with "Fixes #1234 - " where #1234 is the number of the issue.
+* Create a pull request against master.
+
+Please rebase your pull requests on top of the current master using
+  ```git fetch origin && git rebase origin/master``, and squash your changes to a single commit as
```
Comment (gkleiman) [inline docs/docs/contributing.md]: You're missing a closing ` here.

Comment (kolloch) [inline docs/docs/contributing.md]: Thanks, fixed.

**Pull request merged**